src/utils/conversion.ts

```diff
@@ -8,3 +8,8 @@ export function femtoToKilt(big: bigint) {
   const inKilt = big / 10n ** 15n
   return Number(inKilt)
 }
+
+export function microToKilt(big: bigint) {
```

Comment:
Staking rewards are usually pretty low. By limiting them to natural (whole) numbers, users might get frustrated/panicked because they think they are missing out on rewards.